_posts/2019-05-06-technical-writing-for-teams.markdown

```diff
@@ -0,0 +1,198 @@
+---
+layout: epic
+title: "Artsy Writing Office Hours"
+date: "2019-05-06"
+author: [ash]
+categories: [writing, culture]
+---
+
+Welcome back to my series on writing for engineers. In the last post, we discussed [technical writing on the
+web][ash_post] from an individual's perspective: how do you get started? how do you leverage the web? how do you
+improve? In today's post, I want to change directions; instead of focusing on how _individuals_ write, we'll
+discuss how _teams_ write.
+
+<!-- more -->
+
+To get started, I want to return the quote that opened up my first post. When
+[interviewed by the New York Times developer blog](https://open.nytimes.com/five-questions-with-orta-therox-d5bb9659c50b),
+Artsy alumnus [Orta][] said:
+
+> One of my colleagues, Ash Furrow, is really the powerhouse behind improving the state of our public
+> documentation. He runs weekly writing workshops internally and always encourages achievements as being
+> post-worthy. Sometimes the best practice is to have someone who cares encouraging you. It works for me.
+
+This post is going to pick this quote apart, but first I want to address something upfront:
+
+**If you're thinking about helping others write, then you should do it**. Having the desire to help others write is
+the only qualification for _actually helping_. You might think that in order to help others write, you should first
+become a proficient writer yourself. This is backwards. I became the writing powerhouse that Orta mentions _by
+helping others write_.
+
+Let's think about this in terms of coding. When you help teach someone, _you also_ end up learning a lot. Assisting
```

Comment:
#minor - I prefer "_you also_ learn a lot" over "_you also_ end up learning a lot". 

In general, "end up (verb)ing" can usually be replaced with "(verb)" with no loss of message.

Reply:
Yeah, thanks! I think this is a verbal habit of mine that ends up in my writing 😉